.github/workflows/release.yaml

```diff
@@ -42,9 +42,12 @@ jobs:
                   gpg-passphrase: GPG_PASSPHRASE
             - name: Release Habushu
               run: |
+                  git config --local user.email "habushu-noreply@technologybrewery.org"
+                  git config --local user.name "Github Actions"
```

Comment:
A: I'm curious if these config settings like this will persist past the execution? _(if that makes sense)_

Reply:
The settings are supposed to be ephemeral. I can see the "Post Checkout Code" step it runs "/usr/bin/git config --local --unset-all" so it should wipe them out.